tests/libxc/dft-water/input.dat

```diff
@@ -0,0 +1,156 @@
+     'BLYP':   -76.3649513839,        #TEST
+   'HCTH93':   -76.3690959506,        #TEST
+   'M06-HF':   -76.3438580561         #TEST
+# The following functionals are not included 
+# in Psi4 at the moment. 
+#   'PBEOP':   -76.3113060897,        #TEST
+# 'MN12-SX':   -76.3292426233,        #TEST
+#     'GAM':    -76.351646627,        #TEST
+#'MGGA_MS2h':  -76.3874519076,        #TEST
+# 'SOGGA11':   -76.3728240224,        #TEST
+# 'LRC-BOP':   -76.2298882864,        #TEST
+#    'PKZB':   -76.2392402265,        #TEST
+# 'revPBE0':   -76.3469704408,        #TEST
+#  'N12-SX':   -76.3362254975,        #TEST
+#  'revPBE':   -76.3594661693,        #TEST
+# 't-HCTHh':   -76.3688723507,        #TEST
+# 'wM06-D3':   -76.3597621058,        #TEST
+#   'PBE50':   -76.3045269871,        #TEST
+#     'N12':   -76.3313530822,        #TEST
+#   'M06-L':      -76.3773597,        #TEST
+#  'MN12-L':   -76.3121263959,        #TEST
+#  'B3TLAP':   -76.1762459451,        #TEST
+# 'MPW1LYP':   -76.3515538422,        #TEST
+#'SOGGA11-X':  -76.3579662706,        #TEST
+#    'VSXC':   -76.4031806261,        #TEST
+# 'MPW1PBE':   -76.3399258699,        #TEST
+#   'SOGGA':   -75.9775522776,        #TEST
+#    'SCAN':   -76.3525050753,        #TEST
+#'MPW1PW91':   -76.3639997875,        #TEST
+#     'BMK':   -76.3477995384,        #TEST
+#  'MN15-L':   -76.3003777073,        #TEST
+#  'PBEsol':   -76.0376457772,        #TEST
+#   'SCAN0':   -76.3393392953,        #TEST
+#'MGGA_MVS':   -76.3432912834,        #TEST
+#'MGGA_MVSh':  -76.3375136868,        #TEST
+#  'wM05-D':    -76.355280694,        #TEST
+#  't-HCTH':   -76.3695334542,        #TEST
+# 'BP86VWN':   -76.3897099046,        #TEST
+}                                     #TEST
+
+# The following data is from an older 
+# version of Psi4 (~April 2017, SHA: 53e752c)  
+old_psi4_data = {                     #TEST 
+   'BP86': -76.38528296852736,        #TEST
+   'BLYP': -76.36495922440312,        #TEST
+ 'B97-D3': -76.34489583201652,        #TEST
+ 'B3LYP5': -76.34795958728918,        #TEST
+  'wB97X': -76.36677266785263,        #TEST
+    'PBE': -76.29833784290976,        #TEST
+ 'M05-2X': -76.36858376817408,        #TEST
+'wB97X-D': -76.36256087667938,        #TEST
+   'wB97':  -76.3680050769001,        #TEST
+'HCTH120': -76.37568880282406,        #TEST
+   'dlDF':  -76.8428069903893,        #TEST
+   'PBE0': -76.30113068157736,        #TEST
+  'B97-1': -76.36001872648792,        #TEST
+   'PW91': -76.35555307829662,        #TEST
+  'B97-D': -76.34490642131334,        #TEST
+    'M05':  -76.3495817363072,        #TEST
+  'B3LYP': -76.38509296980465,        #TEST
+'HCTH407': -76.37515280900291,        #TEST
+'HCTH147': -76.37958046443578,        #TEST
+  'SOGGA': -75.97737386477169,        #TEST
+  'B97-2': -76.35924563701468         #TEST
+}                                     #TEST
+
+for func in qchem_data.keys():                                                                        #TEST
```

Comment:
Is `qchem_data` a formal superset of psi4 data?

Reply:
Yes, all functionals `old_psi4_data` are in `qchem_data`. However, note that `SOGGA`, which is in `old_psi4_data`, has been commented out in `qchem_data` since it is not currently implemented in Psi4.

Reply:
I've tried implementing `SOGGA`, however there's a problem:

> Water test:
> SOGGA Psi4 dev:       -75.9775599
> SOGGA Psi4 1.0:        -75.9773739
> SOGGA Q-Chem:       -75.9775523

My implementation comfortably passes against Q-Chem, but doesn't pass the tight criterium compared to old Psi4.